README.md

```diff
@@ -154,7 +154,7 @@ Clone recursively to pull-in needed submodule(s):
 
 If you already have a repo cloned, initialize and update:
 
-`$ cd arqma && git checkout release-v0.6.0`    
+`$ cd arqma && git checkout release-v0.6.0.1`    
```

Comment:
It will not be new release but only new TAG inside release-v0.6.0 branch. So that change is not needed